birdhouse/env.local.example

```diff
@@ -309,6 +310,54 @@ export POSTGRES_MAGPIE_PASSWORD=postgres-qwerty
 ## Otherwise, browser have to be closed for culling to work.
 #'--MappingKernelManager.cull_connected=True',
 #])
+#
+#
+# Sample below will allow for sharing notebooks between Jupyter users.
+# Note all shares are public.
+#
+### public-read paths
+#
+## /data/jupyterhub_user_data/public-share/
+#public_read_on_disk = join(jupyterhub_data_dir, 'public-share')
```

Comment:
Not sure I understand why it's called public-share here.

Reply:
Good question !  I choose it because it's a "share" and I want to emphasis on the "public" aspect so user do not put anything sensitive in there so "public-share".

"public-read" is for read-only public stuff that other are sharing with you.

As usually I am terrible with names so please suggest anything else that will make more sense.